pcdsdevices/happi/containers.py

```diff
@@ -346,3 +346,13 @@ class Trigger(Device):
     device_class.default = 'pcdsdevices.device_types.Trigger'
     system = copy(Device.system)
     system.default = 'timing'
+
+
+class MPOD(Device):
```

Comment:
Related - https://github.com/pcdshub/pcdsdevices/issues/665

Reply:
I did not want to add it at first, but I was having issues with testing the creation of `mpod` entries.... because I was having `container=None`... 
Now that i'm thinking about it, I can just use `LCLSItem`  - thank you